inference/core/workflows/core_steps/common/serializers.py

```diff
@@ -143,4 +146,37 @@ def serialise_image(image: WorkflowImageData) -> Dict[str, Any]:
     return {
         "type": "base64",
         "value": image.base64_image,
+        "video_metadata": image.video_metadata.dict(),
     }
+
+
+def serialize_video_metadata_kind(video_metadata: VideoMetadata) -> dict:
+    return video_metadata.dict()
+
+
+def serialize_wildcard_kind(value: Any) -> Any:
+    if isinstance(value, WorkflowImageData):
+        value = serialise_image(image=value)
+    elif isinstance(value, dict):
+        value = serialise_dict(elements=value)
+    elif isinstance(value, list):
+        value = serialise_list(elements=value)
+    elif isinstance(value, sv.Detections):
+        value = serialise_sv_detections(detections=value)
+    return value
+
+
+def serialise_list(elements: List[Any]) -> List[Any]:
```

Comment:
Would be nice to stick to one convention (either seriali`z`e or seriali`s`e)
```suggestion
def serialize_list(elements: List[Any]) -> List[Any]:
```

Reply:
Yeah lets change these to z everywhere.  Can be a separate PR if there is a lot of them.